HOWTO.md

```diff
@@ -0,0 +1,75 @@
+libsbp Development Procedures
+=============================
+
+This document summarizes some practices around contributions to this
+library. These instructions don't come with a warranty yet, so please
+feel free to update it to mirror reality.
+
+# Adding and Testing New Messages
+
+Adding new SBP messages is currently a very organic, social
+process. This may change in the future.
+
+0. Read, understand, and imitate the current SBP definition syntax by
+   looking at the annotated [`example`](spec/example/yaml) and
+   [`existing`](spec/yaml/swiftnav/sbp) messages.
+
+1. Add a message definition to the approprate package, or create a new
+   one if needed. The message ID needs to be unique. A message should
```

Comment:
I think this should reference a section "Message Guidelines". I have a few more things that would go under guidelines, I'll follow up in another comment.